CMakeLists.txt

```diff
@@ -81,8 +81,12 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 include(GetGitVersion)
 get_git_version(GIT_VERSION)
 
+if ("${GIT_VERSION}" STREQUAL "v0.0.0")
+  set(VERSION "${benchmark_VERSION}")
+else()
+  string(REGEX MATCH "[0-9]+\\.[0-9]+\\.[0-9]+" VERSION ${GIT_VERSION})
+endif()
```

Comment:
This is fine if it works on it's own.

Reply:
No, it doesn't work on its own, because when there's no version info from git, it has no way of knowing what to set VERSION to.

Reply:
Close:(Google:master)

Reply:
Close:(cla:no)